app/controllers/application_controller.rb

```diff
@@ -3,11 +3,31 @@ class ApplicationController < ActionController::Base
 
   def set_nav
     @nav = [
-      { text: "Home", path: dashboard_path, icon: "bi bi-house-door" },
-      { text: "Search", path: search_path, icon: "bi bi-search" },
-      { text: "New Item", path: new_item_path, icon: "bi bi-plus-square" },
-      { text: "Notifications", path: notifications_path, icon: "bi bi-bell" },
-      { text: "Profile", path: profile_path, icon: "bi bi-person" }
+      { text: t("views.home.title"), path: dashboard_path, icon: "bi bi-house-door" },
+      { text: t("views.search.title"), path: search_path, icon: "bi bi-search" },
+      { text: t("views.new_item.title"), path: new_item_path, icon: "bi bi-plus-square" },
+      { text: t("views.notifications.title"), path: notifications_path, icon: "bi bi-bell" },
+      { text: t("views.profile.title"), path: profile_path, icon: "bi bi-person" }
```

Comment:
@Nils1729 Is there a reason as to why you changed my changes here? The reason that I did not call the translate function here is that this does not work since it is not properly scoped to the actual request. Your solution results is only an english translation. 

Can you please find a different solution or revert to my solution? 
![image](https://user-images.githubusercontent.com/16378455/204108104-cc41cd46-c9a0-42ef-942d-ed23cd1d9c4c.png)

Reply:
I did that change because I'd like to avoid 'raw' identifiers to be passed around in the application whenever possible. You can change it back if you want.

Reply:
I agree that the current solution isn't exactly the nicest, however, I do not see an alternative at this point. I changed this back to the version I had and updated the identifiers to match your schema.